Source/Immutable/Public/Immutable/ImmutablePassport.h

```diff
@@ -163,89 +234,274 @@ class IMMUTABLE_API UImmutablePassport : public UObject
+	 */
 	void OnConnectPKCEResponse(FImtblJSResponse Response);
+
+	/*
+	 * Callback when deep link is activated.
+	 *
+	 * @param DeepLink The deep link URL that was activated. 
+	 */
 	void OnDeepLinkActivated(FString DeepLink);
-    void CompleteLoginPKCEFlow(FString Url);
+
+	/*
+	 * Completes the PKCE login flow using the provided URL.
+	 * 
+	 * @param Url The URL containing the authorisation code and state.
+	 */
+	void CompleteLoginPKCEFlow(FString Url);
 #endif
 
 #if PLATFORM_ANDROID
+	/**
+	 * Callback when Login PKCE is dismissed.
+	 */
 	void HandleOnLoginPKCEDismissed();
+
+	/**
+	 * Calls a static void method in Java using JNI.
+	 *
+	 * @param Env		The JNI (Java Native Interface) environment.
+	 * @param Class		The Java class containing the method.
+	 * @param Method	The method ID of the method to call.
+	 * @param ...		Additional parameters to forward to the method to call.
+	 */
 	void CallJniStaticVoidMethod(JNIEnv* Env, const jclass Class, jmethodID Method, ...);
+
+	/**
+	 * Launches a URL on Android using JNI.
+	 *
+	 * @param Url The URL to launch.
+	 */
 	void LaunchAndroidUrl(FString Url);
 #endif
 
+	/**
+	 * Sets the specified state flags by applying a bitwise OR operation.
+	 * @param StateIn The state flags to be set.
+	 */
 	void SetStateFlags(uint8 StateIn);
+
+	/**
+	 * Resets the specified state flags by applying a bitwise AND operation with the negated flags.
+	 * @param StateIn The state flags to be reset.
+	 */
 	void ResetStateFlags(uint8 StateIn);
+
+	/**
+	 * Checks if the specified state flags are set. 
+	 * @param StateIn The state flags to check.
+	 * @return True if all StateIn flags are set, otherwise, false.
+	 */
 	bool IsStateFlagsSet(uint8 StateIn) const;
 
 protected:
+	/** Cached pointer to the JavaScript connector used for communicating JavaScript calls. */
 	TWeakObjectPtr<UImtblJSConnector> JSConnector;
+	/** Cached passport init data. */
 	FImmutablePassportInitData InitData;
 	FDelegateHandle BridgeReadyHandle;
+	/** A map of JavaScript calls request Ids to their response callbacks. */
 	TMap<FString, FImtblPassportResponseDelegate> ResponseDelegates;
 
 #if PLATFORM_ANDROID
+	/** Delegate called when the PKCE flow is dismissed. */
 	FImtblPassportOnPKCEDismissedDelegate OnPKCEDismissed;
 #endif
 
 #if PLATFORM_ANDROID | PLATFORM_IOS | PLATFORM_MAC
+	/** Delegate for handling deep link activation. */
 	FImtblPassportHandleDeepLinkDelegate OnHandleDeepLink;
 	// Since the second part of PKCE is triggered by deep links, saving the
 	// response delegate here so it's easier to get
 	FImtblPassportResponseDelegate PKCEResponseDelegate;
+	/** Delegate for handling PCKE logout. */
 	FImtblPassportResponseDelegate PKCELogoutResponseDelegate;
 	// bool IsPKCEConnected = false;
 #endif
 
 
 private:
+	/**
+	 * Saves the current passport settings to save game object.
+	 */
 	void SavePassportSettings();
+
+	/**
+	 * Loads the passport settings from save game object.
```

Comment:
Might want to keep it consistent, sometimes we use small p for Passport, sometimes capital p.
I'd go with a capital P.